core/domain/event_services.py

```diff
@@ -65,6 +65,75 @@ def record(cls, *args, **kwargs):
         cls._handle_event(*args, **kwargs)
 
 
+class StatsEventsHandler(BaseEventHandler):
+    """Event handler for incremental update of analytics model using batched
+    event models.
```

Comment:
It's just batched event dicts, right? Models are what we have in the GAE storage layer, but I don't think this touches that (as far as the events themselves are concerned).

Reply:
Done.